Resources/Prototypes/_RMC14/Roles/Ranks/Colony/civilan.yml

```diff
@@ -0,0 +1,4 @@
+﻿- type: rank
+  id: RMCRankCivilan
+  name: Civilan
```

Comment:
```suggestion
  name: Civilian
```

On mobile so can't find if there's any other instance of this

Reply:
thanks